wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -319,21 +321,32 @@ def table_header_label(context, label=None, sortable=True, ordering=None, sort_c
         # currently ordering forwards on this column; link should change to reverse ordering
         url = querystring(context, **{sort_param: reverse_sort_field})
         classname = "icon icon-arrow-down-after teal"
+        ordering_label = _('descending')
 
     elif ordering == reverse_sort_field:
         # currently ordering backwards on this column; link should change to forward ordering
         url = querystring(context, **{sort_param: sort_field})
         classname = "icon icon-arrow-up-after teal"
+        ordering_label = _('ascending')
 
     else:
         # not currently ordering on this column; link should change to forward ordering
         url = querystring(context, **{sort_param: sort_field})
         classname = "icon icon-arrow-down-after"
+        ordering_label = _('ascending')
+
+    title_tag = _('Sort the order of child pages within \'')
+    title_tag += parent_page_title
+    title_tag += "%s" % (_('\' by \''))
+    title_tag += label
+    title_tag += "%s" % (_('\' in '))
+    title_tag += "%s" % (ordering_label)
+    title_tag += "%s" % (_(' order.'))
```

Comment:
From what I understand this is going to output `by`, `in`, `order` as separate strings to be translated, then concatenated again. This is problematic because the parts of the sentence won’t necessarily be in the same order in other languages.

Instead, we want the Python equivalent of `blocktrans`, where the translators get given a single string with placeholders for dynamically-injected content. I think this would look like this:

```python
title_tag_context = {
    'parent': parent_page_title,
    'label': label,
    'ordering: ordering_label,
}
title_tag = _('Sort the order of child pages within \'%(parent)\' by \'%(label)\' in %(ordering) order.') % 
```

Reply:
The issue with translating it all together is that the label has already been translated in the template. I guess I could just remove the translation of the label in the template, and then translated it whereever it is output in the template.

Reply:
Will await @gasman's response below before updating this any further.

Reply:
Hi @helenb, a slightly cleaner variation of this in python would be:
```
title_tag = _("Sort the order of child pages within '{parent}' by '{label}' in {ascending_or_descending} order.").format(
    parent=parent_page_title,
    label=label,
    ascending_or_descending=ordering_label,
)

Reply:
Now addressed in fbc3f922bc240506a27aa297cb96e3f2d6402d85.